src/res/fallback.rs

```diff
@@ -0,0 +1,111 @@
+#[cfg(feature = "nightly")]
+use std::intrinsics::type_name;
+
+use {Fetch, FetchMut, Resource, ResourceId, Resources};
+
+#[cfg(feature = "nightly")]
+macro_rules! fetch_panic {
+    () => {
+        {
+            panic!(
+                "Tried to fetch a resource of type {:?}, but the resource does not exist.\n\
+                 Try adding the resource or \
+                 using `Option<Fetch>` / `Fetch` instead of `FetchExpect`.",
+                unsafe { type_name::<T>() },
```

Comment:
Please, add name of `T` as argument to the macro (`$t:ty`) instead of relying on type parameter being name `T`.

Reply:
It's local, so this isn't necessary.